### DISCUSSION TITLE
cannot open file 'lib\Debub\axmol.lib' and Cannot open include file : 'WevView2.h'

### DISCUSSION THREAD

**Discussion by rudiHammad** [axmolengine/axmol#1044] [Q&A] [answered]
Hello,
today I tried for the first time to switch from cocos2dx 4 to axmol. I followed the install instructions. I was able to succesfully build the 
visual studio solution for the tests in the axmol folder and run them okey.
But when a I tried to create a new visual studio solution i get the errors mentioned in the title

![Capture](https://user-images.githubusercontent.com/25636976/216772170-1191a097-35ee-42af-bca4-a70c08aeb5d6.JPG)

I am on windows 10 and visual studio 2022 comunity. I have run setup.py, created the project with axmol new -p YOUR.UNIQUE.ID -d PROJECT_PATH -l cpp, and creatd the visual studio solution with cmake -S . -B build -G "Visual Studio 17 2022" -A x64.

(Just in case, I have axmol installed in C: drive and my game project in a D: drive, but that should have nothing to do with it right?)

Thanks,
R

**Comment by DelinWorks** [comment]
WebView2 Prevents the engine from compiling because the engine failed to find WebView2 in your windows environment, You need to install the WebView2 Extension on your windows machine ([WebView2 Download Page](https://developer.microsoft.com/en-us/microsoft-edge/webview2/)) or If that doesn't work out for you because of a windows bug like I had or you don't need the WebView2 Extension you can specify for the engine NOT to include it in the build by removing `_AX_HAVE_WEBVIEW2` from preprocessor flags in the engine solution. Hope that helps!

**Comment by rh101** [comment, marked as answer]
Add this line to your CMakeLists.txt file after the `project(${APP_NAME})` line:

`set(AX_ENABLE_MSEDGE_WEBVIEW2 OFF CACHE BOOL "Enable msedge webview2" FORCE)`

It should look something like this:
```
project(${APP_NAME})

set(AX_ENABLE_MSEDGE_WEBVIEW2 OFF CACHE BOOL "Enable msedge webview2" FORCE)

if(NOT DEFINED BUILD_ENGINE_DONE) # to test demogame into root project
    if(XCODE)
...
etc etc
...
```

Make sure you either delete the build folder you created with cmake, or go into the build folder and delete the `CMakeCache.txt`, then re-run the same cmake command again that you used to create the build folder, otherwise the `AX_ENABLE_MSEDGE_WEBVIEW2 ` setting may not work.

Also, copy the console output from cmake and paste it in a post on here.  There may be clues in there as to what might be causing your issue.

**Reply by rudiHammad** [reply]
Great, that works. I forgot to rebuild the "build" folder after changing the  CMakeLists.txt.
Thanks for all the replies.

**Reply by aismann** [reply]
@rudiHammad 
Please close your question also here plz, thanks:
https://discuss.cocos2d-x.org/t/can-someone-provide-a-working-shader-code-for-cocos2dx-4/58181

**Reply by rudiHammad** [reply]
sure, done.

**Reply by codehappyness** [reply]
Thank for help
